samples/client/petstore/python/Makefile

```diff
@@ -1,21 +0,0 @@
- #!/bin/bash
```

Comment:
We need this file for testing, please add it back

Reply:
Done

Reply:
Can you resolve this conversation?